daemon/src/db.rs

```diff
@@ -39,6 +42,15 @@ mod failed;
 
 pub mod time_to_first_position;
 
+const MAINNET_MAKER_ID: &str = "7e35e34801e766a6a29ecb9e22810ea4e3476c2b37bf75882edf94a68b1d9607";
+const MAINNET_MAKER_PEER_ID: &str = "12D3KooWP3BN6bq9jPy8cP7Grj1QyUBfr7U6BeQFgMwfTTu12wuY";
+
+const TESTNET_MAKER_ID: &str = "69a42aa90da8b065b9532b62bff940a3ba07dbbb11d4482c7db83a7e049a9f1e";
+const TESTNET_MAKER_PEER_ID: &str = "12D3KooWEsK2X8Tp24XtyWh7DM65VfwXtNH2cmfs2JsWmkmwKbV1";
+
+const DEV_MAKER_ID: &str = "10d4ba2ac3f7a22da4009d813ff1bc3f404dfe2cc93a32bedf1512aa9951c95e";
+const DEV_MAKER_PEER_ID: &str = "12D3KooWDjzHna3pNi1Bt1DoRfrpsBREykJKXDRDxXvhJNAdDZEk";
```

Comment:
❓ I guess we don't want to merge this?

Reply:
Why not? 
We don't have to but all of these are temporary until we remove the old connection code. Since we know that one we we thought about adding it.

Reply:
It would be good if this lookup was contained inside a function, so that we don't need to expose these constants globally (so it's obvious why they're there) - see other comment. If we won't be able to use the const for the tests, it might be enough to just ensure it's not `None` when a lookup works. 
We'll test all 3 combinations anyway.

Reply:
Happy to do so, but then I have to hardcode the values in the tests instead of using the constants.

Reply:
The ones I was wondering about specifically were the ones with the `DEV_` prefix. What are they about?

Reply:
I removed them because I actually think what @klochowicz and me had in mind does not really make sense. 
If we open a new CFD using our shared dev seed then that new CFD will have a peer-id anyway. I don't think we have to support rolling over old `Cfd` with the new protocol in the dev environment.